File: 백준/Bronze/2525. 오븐 시계/오븐 시계.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
let [시간, 분] = input[0].split(' ').map((v) => Number(v));

분 += Number(input[1]);
if (분 >= 60) {
  시간 += Math.floor(분 / 60);
  분 = 분 % 60;
}
if (시간 >= 24) {
  시간 = 시간 % 24;
}

console.log(시간, 분);

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
A, B = input().split()
A = int(A) # 시간
B = int(B) # 분
C = int(input())
D = C//60 # 시간
E = C%60 # 분

H = A+D
M = B+E
if(M>=60):
    H = H+(M//60)
    M = M%60
if(H>=24):
    H = H%24

print(str(H)+" "+str(M))